proto.md

```diff
@@ -271,27 +309,70 @@ Flüstern
 
 		C: whisper <Spieler> <Nachricht>
 		S: +JAWOHL
-		
 		S: chat-update <Nachricht>
 		C: +JAWOHL
 		
 #### Beschreibung
 
-Der `whisper`-Befehl ermöglicht es einem Spieler, einen anderen direkt anzuschreiben, ohne dass dies von den anderen Spielern bemerkt wird.
+Der `whisper`-Befehl ermöglicht es einem Spieler, einen anderen direkt anzuschreiben,
+ohne dass dies von den anderen Spielern bemerkt wird.
 
-Errorhandling
--------------
+Ereigniskarte ziehen
+--------------------
 
-Tritt ein Fehler bei einem Befehl auf, wird dies meist durch einen Fehlercode quittiert.
+#### Synopsis
 
-Diese sind von den `SMTP reply codes` inspiriert und werden wie folgt gebildet:
+		S: eventcard <Text>
+		C: +JAWOHL
+		oder
+		C: -NEIN 133 Please gimme a different card!
+		S: JAWOHL
+		oder
+		C: -NEIN 133 Please gimme a different card!
+		S: -NEIN 134 You cannot take a different card.
+		C: -JAWOHL
+
+#### Beschreibung
+
+`eventcard` wird nur an den Spieler gesendet, der gerade eine Ereigniskarte ziehen musste.
+Enthalten ist nur der Text der Karte; die Aktion wird, sofern möglich, vom Server sofort
+durchgeführt. Falls man die Möglichkeit hat, statt der Aktion eine andere Karte zu ziehen
+("Tun Sie das und das oder nehmen Sie eine Gemeinschaftskarte"), returnt der Client mit
+einem `-NEIN 133`. Fals das nicht möglich ist, beschwert sich der Server, was der Client
+akzeptieren muss.
+
+Errorhandling
+-------------
 
-	Erste Stelle: Wie schwerwiegend ist der Fehler? (Aufgrund von `+JAWOHL` benötigen wir hier keine positiven Werte.)
-		1yz  Vorübergehendes Fehlschlagen (Der selbe Befehl kann zu einem anderen Zeitpunkt funktionieren.)
-		2yz  Permanentes Fehlschlagen (Dieser Befehl wird so nie funktionieren.)
-	Zweite Stelle: Wo ist der Fehler aufgetreten?
-		x0z  Allgemeine Fehler die in verschiedenen Bereichen auftreten können
-		x1z  Beitreten, Chatupdate
-		x2z  Chatten, Flüstern
-	Dritte Stelle: Was genau ist passiert?
-		Die Bedeutung der dritten Stelle hängt von der zweiten Stelle ab.
+Tritt ein Fehler bei einem Befehl auf, wird dies *meist* durch einen Fehlercode quittiert;
+diese sind von den *SMTP Reply-Codes* inspiriert. Aufgrund von `+JAWOHL` benötigen wir hier
+keine positiven Werte.
+
+1. Ziffer: Dauerhaftigkeit des Fehlers
+2. Ziffer: Ort des Fehlers
+3. Ziffer: genauere Einteilung
+
+Code | Beschreibung
+-----|-------------
+ **1yz** | vorübergehendes Fehlschlagen; der Befehl kann zu einem anderen Zeitpunkt funktionieren.
+ **13z** | temp. Gameplay-Fehler
+ 131 | nicht genug Geld
+ 132 | keine Gefängnis-Frei-Karte vorhanden, die benutzt werden kann
+ 133 | Client will eine neue Ereigniskarte ziehen
```

Comment:
For all intents and purposes, they are the same except that they are on different stacks and are taken on different occasions. I know what you mean (*"... und ziehen Sie eine Gemeinschaftskarte"*); the server knows, too, because it knows the card has this scripted feature; the player knows via the message; the protocol and the client does **not** need to know.